### PR TITLE
Correct "our community forum" link

### PR DESCRIPTION
**Pull request by SuriyaaKudoIsc**
@pathawks @parkr 

New pull request of #245!

### CODE REVIEW

**Comment by parkr** [comment]
@SuriyaaKudoIsc Could you change the text from "our help repo" to "our community forum"

**Comment by SuriyaaKudoIsc** [comment]
:+1: Done!

File: History.markdown [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
   * Drupal 6 importer depends on mysql. (#242)
 
+### Site Enhancements
+
+  * Correct "How to Contribute" link (#244)
+
```
Comment (parkr) [inline History.markdown]: Please remove this – this is added automatically by @jekyllbot on merge.

Comment (parkr) [inline History.markdown]: Oh, this is the Jekyllbot commit. How strange that it's in this PR.

**Comment by parkr** [comment]
@jekyllbot: merge +site

**Pull request merged**